public/static/locales/en/my-forms.json

```diff
@@ -58,9 +58,11 @@
     "submissionID": "Submission ID",
     "copyResponse": "Copy answer row to clipboard"
   },
-  "protectedBWarning": {
+  "dataClassification": {
     "officialRecord": "This is an official record",
     "retainCopy": "Retain this copy in accordance to your retention policy.",
+    "unclassified": "UNCLASSIFIED",
```

Comment:
Should these be all uppercase?

We use `Protected A` etc... on other spots

Reply:
In the Figma file we can see that the data classification label is written in full uppercase